netbox/extras/scripts.py

```diff
@@ -470,6 +470,14 @@ def _run_script():
         except AbortTransaction:
             script.log_info("Database changes have been reverted automatically.")
             clear_webhooks.send(request)
+        except AbortScript as a:
```

Comment:
This is a _super_ minor nit, but could you name the captured exception `e` rather than `a`? This is purely a convention thing.

```suggestion
        except AbortScript as e:
```